recipe/conda_build_config.yaml

```diff
@@ -3,3 +3,5 @@ pytorch_version:
   - 1.9.0
   - 1.9.1
   - 1.10.0
+channel_targets:
+- conda-forge openmm_rc
```

Comment:
Shouldn't this be `openmm-torch_rc`

Reply:
We need all the packages to share the same label.  Someone should ultimately be able to type

```
conda install -c conda-forge/label/openmm_rc openmm-ml
```

and it will automatically install coordinated packages for OpenMM, NNPOps, OpenMM-Torch, and OpenMM-ML.

Reply:
That should work, I won't tell anyone but I think the right way to do that is with `channel_sources` https://github.com/conda-forge/cfep/blob/main/cfep-05.md#specification since technically the spec doesn't seem to indicate the tags should be used that way. I could be wrong though!

Reply:
As far as I can tell from the documentation, `channel_sources` only applies at build time.  To install, a user would have to enter something really complicated like

```
conda install -c conda-forge/label/openmm_rc -c conda-forge/label/nnpops_rc -c conda-forge/label/openmm-torch_rc -c conda-forge/label/openmm-ml_rc openmm-ml
```

Reply:
Yes, I agree that it is at build time, and users would have to type that gross line.

Reply:
I am afraid one label per project is necessary. Otherwise people won't be able to opt-in for the pre-release version of individual packages. Please, consider following CFEP-5 as intended.

Reply:
What's the intended way of handling cases like this, where you have coordinated pre-release versions of a bunch of connected packages?

Reply:
I am not aware of such a situation.

Reply:
@peastman We could create a cloud environment so users could do something like `conda env create openmm/opemm-ml-beta` to make it easier to try out if your worried about that.

Reply:
This package is an example of that situation.  It's very common for a single project to be split into multiple conda packages.  Check out Sphinx, PyTorch, TensorFlow, etc.  I think the only reason this issue doesn't come up all the time is that those projects don't use conda-forge as their main distribution channel, so pre-releases are never posted to conda-forge.

Since there's no existing policy for this situation, can we create one?  All we need is to use the same label for all the prerelease packages that need to go together.

Reply:
> We could create a cloud environment so users could do something like conda env create openmm/opemm-ml-beta to make it easier to try out if your worried about that.

That wouldn't address the problem.  We need people testing the beta with their real projects on their real hardware.

Reply:
@peastman It would let users install all of the beta/rc packages in a single and simple line. By "cloud" I meant that the env file is hosted on anaconda, but they would use that command to install the packages locally. So instead of 
`conda install -c conda-forge/label/openmm_rc -c conda-forge/label/nnpops_rc -c conda-forge/label/openmm-torch_rc -c conda-forge/label/openmm-ml_rc openmm-ml openmm nnpops openmm-torch openmm-ml`
it would be `conda env create openmm/opemm-ml-beta`
So then 1) it will be easy for people to try the packages and 2) follows CFEP-5